erpnext/stock/get_item_details.py

```diff
@@ -469,18 +469,18 @@ def get_item_tax_info(company, tax_category, item_codes, item_rates=None, item_t
 		item_tax_templates = json.loads(item_tax_templates)
 
 	for item_code in item_codes:
-		if not item_code or item_code[1] in out or not item_tax_templates.get(item_code[1]):
+		if not item_code or item_code in out or not item_tax_templates.get(item_code):
 			continue
 
-		out[item_code[1]] = {}
-		item = frappe.get_cached_doc("Item", item_code[0])
```

Comment:
This line tries to fetch something that doesn't exist and hence the 404, I believe. Feel free to correct if I'm wrong.

Reply:
I think in that case either of these two exceptions would be raised.
1. 'type(item)' is not subscriptable
2. index is out of range

Reply:
No, item_code is a string. Therefore, item_code[0] returns the first character.
As visible in #26136, the item_code is FG-00022, but on the popup, it says Item F.

Reply:
Line 476 becomes `item = frappe.get_cached_doc("Item", "F")` then. And there is no item called "F" in the system and hence 404.

Reply:
Oh, I understood. Thanks a lot.

Reply:
Happy to help, thank you for contributing :100:

Reply:
Also try to follow these conventions for your commit messages - https://github.com/pvdlg/conventional-commit-types